packages/browser/src/defaultSession.ts

```diff
@@ -119,8 +123,18 @@ export const onLogout: Session["onLogout"] = (...args) => {
  *
  * @param callback The function called when a session is restored.
  * @since 1.3.0
+ * @deprecated Prefer events.on(EVENTS.SESSION_RESTORED, callback)
  */
 export const onSessionRestore: Session["onSessionRestore"] = (...args) => {
   const session = getDefaultSession();
   return session.onSessionRestore(...args);
 };
+
+/**
+ * {@link SessionEventEmitter} instance to subscribe to events by the default session.
+ *
+ * @since Unreleased
+ */
+export const events = (): Session["events"] => {
+  return getDefaultSession().events;
+};
```

Comment:
What is the rationale for making this a function of its own rather than having consumers just use the `.events` property of the default session?